scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -55,6 +55,18 @@
   }
 }
 
+.nav-pills {
+  margin-bottom: 15px;
```

Comment:
[navs](http://getbootstrap.com/components/#nav-pills) in Bootstrap don't have a bottom margin